ArduCopter/control_auto.cpp

```diff
@@ -324,6 +324,13 @@ void Copter::auto_land_run()
 
     // process pilot's input
     if (!failsafe.radio) {
+        if((g.throttle_behavior & THR_BEHAVE_HIGH_THROTTLE_CANCELS_LAND) != 0 && rc_throttle_control_in_filter.get() > 0.7f*THR_MAX){
+            // exit land if throttle is high
+            if (!set_mode(LOITER)) {
```

Comment:
Might be a style thing but I honestly find that less clear.
On Jan 7, 2016 14:47, "Tom Pittenger" <notifications@github.com> wrote:

> In ArduCopter/control_auto.cpp
> <https://github.com/diydrones/ardupilot/pull/3442#discussion_r49130555>:
>
> > @@ -324,6 +324,13 @@ void Copter::auto_land_run()
> >
> >      // process pilot's input
> >      if (!failsafe.radio) {
> > +        if((g.throttle_behavior & THR_BEHAVE_HIGH_THROTTLE_CANCELS_LAND) != 0 && rc_throttle_control_in_filter.get() > 0.7f*THR_MAX){
> > +            // exit land if throttle is high
> > +            if (!set_mode(LOITER)) {
>
> For clarity, this should read:
> bool success = set_mode(LOITER);
> if (!success) { set_mode(ALT_HOLD); }
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot/pull/3442/files#r49130555>.
>